core/common/src/test/java/alluxio/collections/DirectedAcyclicGraphTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.collections;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Tests the {@link DirectedAcyclicGraph} class.
+ */
+public final class DirectedAcyclicGraphTest {
+  private DirectedAcyclicGraph<Integer> mGraph;
+
+  /**
+   * Initialize.
+   */
+  @Before
+  public void before() {
+    mGraph = new DirectedAcyclicGraph<>();
+  }
+
+  /*
+   * Tests some simple graphs.
+   */
+  @Test
+  public void simpleGraphTest() {
+    // Empty graph.
+    Assert.assertTrue(mGraph.getRoots().isEmpty());
+    Assert.assertTrue(mGraph.sortTopologically(new HashSet<Integer>()).isEmpty());
+
+    // One node.
+    mGraph.add((Integer) (1), new ArrayList<Integer>());
+    Assert.assertEquals(mGraph.getRoots().size(), 1);
```

Comment:
I think for `assertEquals()` the expected value should be first, and the actual value you are testing comes second. If so, please update all the relevant asserts to that parameter ordering.